src/components/EditButton/EditButton.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { BrowserRouter as Router } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import { EditButton } from ".";
+
+describe("EditButton tests:", () => {
```

Comment:
you don't must to duplicate the tests, just a few is fine but it could be a little more extended

Reply:
Ok